docs/DeveloperGuide.md

```diff
@@ -18,21 +599,21 @@
 
 {Describe the value proposition: what problem does it solve?}
 
-## User Stories
+## Appendix B: User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ... | I want to ...             | So that I can ...                                           |
+|---------|----------|---------------------------|-------------------------------------------------------------|
+| v1.0    | new user | see usage instructions    | refer to them when I forget how to use the application      |
+| v2.0    | user     | find a to-do item by name | locate a to-do without having to go through the entire list |
 
-## Non-Functional Requirements
+## Appendix C: Non-Functional Requirements
 
 {Give non-functional requirements}
 
-## Glossary
+## Appendix D: Glossary
 
 * *glossary item* - Definition
```

Comment:
Do add definitions to your glossary to help the user understand the terms you've used.